pkg/ec2pricing/ec2pricing.go

```diff
@@ -48,7 +48,8 @@ type spotPricingEntry struct {
 // New creates an instance of instance-selector EC2Pricing
 func New(sess *session.Session) *EC2Pricing {
 	return &EC2Pricing{
-		PricingClient: pricing.New(sess),
+                // use us-east-1 since pricing only has endpoints in us-east-1 and ap-south-1
```

Comment:
outside the scope of this PR, but interested to know latency differences for "worst" case. First that comes to mind is a user in ap-south-1 (Mumbai) using PricingClient with `us-east-1` region.

Based on that data, we wouldn't need to make this value configurable, but could auto-assign the value based on user's aws credential region (if the time savings aren't negligible).

Reply:
yeah... I definitely see your point here. It's not too time sensitive data since this is mainly used as a CLI, but let's defer until an issue arises.